src/app.js

```diff
@@ -51,10 +51,10 @@ const App = () => {
       <nav className="pane resourceLinks">
         <h2>Resources</h2>
         <ul className="scrollable scrollable_y">
-          {Object.keys(resourceLinks).map((type, index) => (
-            <li key={`resource-link-${index}`}>
-              <Link title={type} url={resourceLinks[type].href} handleClick={updateDocument} />
-            </li>
+          {Object.keys(resourceLinks).map((key, index) => (
+              <li key={`resource-link-${index}`}>
+                <LinkElement link={resourceLinks[key]} handleClick={updateDocument} />
```

Comment:
I'm glad you renamed this, besides the new class you made <Link> was not a good component name

Reply:
*link* was not a good component name